docs/src/content/docs/cookbook/jettons.mdx

```diff
@@ -360,6 +360,48 @@ fun calculateJettonWalletAddress(
 }
 ```
 
+### Onchain metadata creation
+
+```tact
+/// https://github.com/ton-blockchain/TEPs/blob/master/text/0064-token-data-standard.md#jetton-metadata-example-offchain
+fun composeJettonMetadata(
+    name: String,        // full name
+    description: String, // text description of the Jetton
+    symbol: String,      // "stock ticker" symbol without the $ prefix, like USDT or SCALE
+    image: String,       // link to the image
+    // There could be other attributes, see:
```

Comment:
I would name those "fields" not "attributes" because attributes has slightly different meaning in terms of NTFs